recipes/depot_tools/all/conanfile.py

```diff
@@ -1,24 +1,36 @@
 import os
 import shutil
-from conans import ConanFile, tools
+
+from conan import ConanFile
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get
+from conan.tools.layout import basic_layout
+
+required_conan_version = ">=1.52.0"
 
 
 class DepotToolsConan(ConanFile):
     name = "depot_tools"
-    url = "https://github.com/conan-io/conan-center-index"
-    homepage = "https://chromium.googlesource.com/chromium/tools/depot_tools"
     description = "Tools for working with Chromium development."
-    topics = ("depot_tools", "chromium")
     license = "BSD-3-Clause"
-    short_paths = True
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://chromium.googlesource.com/chromium/tools/depot_tools"
+    topics = ("chromium", "pre-built")
+
+    package_type = "application"
+    settings = "os", "arch", "compiler", "build_type"
     no_copy_source = True
-    settings = "os", "arch", "build_type", "compiler"
-    exports_sources = ["patches/**"]
+    short_paths = True
 
+    def export_sources(self):
+        export_conandata_patches(self)
 
-    @property
-    def _source_subfolder(self):
-        return os.path.join(self.source_folder, "source_subfolder")
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def package_id(self):
+        del self.info.settings.arch
```

Comment:
How is arch deleted from an application? That seems like it's going to break super easy no?

Reply:
This project consists of Python scripts only, so it should not be dependent on architecture. Looking at it again, `self.info.clear()` might be even more appropriate here. I don't think it's os-dependent either.

Reply:
Ohhh, that makes a lot of sense!